docs/src/pages/guides/examples.md

```diff
@@ -17,7 +17,7 @@ If you didn't define the http method, it will use the GET method by default. Ple
 
 ## Enabling Notification
 
-Here is a probe example to monitor Monika landing page:
+Probes are more useful with notifications enabled to alert when something is amiss. Below is a probe example to monitor Monika landing page with an email notification:
```

Comment:
Instead of `Below is a probe example to monitor Monika landing page with an email notification`, i think it's better to be `The following is a configuration example to get notification via e-mail when incident happens.`